jenkins/Dockerfile-blossom.ubuntu

```diff
@@ -42,4 +46,7 @@ RUN update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64
 RUN ln -s /usr/bin/python3.8 /usr/bin/python
 RUN python -m pip install pytest sre_yield requests pandas pyarrow findspark pytest-xdist pre-commit
 
-RUN apt install -y inetutils-ping expect
+# libnuma1 and libgomp1 are required by ucx packaging
+RUN apt install -y inetutils-ping expect wget libnuma1 libgomp1
+
+RUN mkdir -p /tmp/ucx && cd /tmp/ucx && wget https://github.com/openucx/ucx/releases/download/${UCX_VER}/ucx-${UCX_VER}-ubuntu${UBUNTU_VER}-mofed5.x-cuda${CUDA_VER}.deb && dpkg -i *.deb && rm -rf /tmp/ucx
```

Comment:
nit:  line too long, newline for line continuation would be better~

Reply:
long URL, but https://github.com/NVIDIA/spark-rapids/pull/3218/commits/210d7ea713401602fba57139243c661c4b1b91fe breaks it into 1 line per command.